modules/router/templates/router_include.conf.erb

```diff
@@ -51,7 +51,9 @@ location ~ ^/performance/ {
   proxy_pass http://varnish;
 }
 
-location ~ ^/contact/govuk/(service-feedback|problem_reports)$ {
+# This list of URLS should be kept in sync with the POST-able routes in the
```

Comment:
You mean this comment won't do it on it's own? ;)

Good question, I have no idea - is there a good place to document it?  I guess something in the README for the feedback app would be a good start.